ScoutSuite/providers/gcp/resources/projects.py

```diff
@@ -0,0 +1,18 @@
+from ScoutSuite.providers.gcp.resources.resources import GCPCompositeResources
+
+class Projects(GCPCompositeResources):
+    def __init__(self, gcp_facade):
```

Comment:
Can you type the facade?

Reply:
I had it in another branch but apparently I forgot to add it in this one, good catch!